### PR TITLE
Add @yframe directive

### PR DESCRIPTION
**Pull request by RoduanKD**
Peace be upon you

Thank you for the efforts you've put to make this cool package.

I've added a new way to use this package by using a directive with will make using this package a lot easier for future users.
example usage
```php
@yframe('https://www.youtube.com/watch?v=XXXXXXX', ['width' => '400px'])
```

thanks again

### CODE REVIEW

**Review by karam-mustafa — CHANGES_REQUESTED** [review]
You must add a comment block before each function, explaining its work and the things it returns, and the owner of these changes, you can see the existing functions to see how we add comments before each function

File: src/Providers/YFrameServiceProviders.php [hunk 1]
```diff
@@ -64,4 +66,33 @@ private function resolveCommands()
             ]);
         }
     }
+
+    protected function registerDirectives()
```
Comment (karam-mustafa) [inline src/Providers/YFrameServiceProviders.php:76]: you must add a comment block here

**Comment by karam-mustafa** [comment]
Hi Roduan, Thanks for your contributions, I just want to mention that it will be great if you write a test for the @yframe directive, and you can add it as a feature test.
Thanks again

File: src/Providers/YFrameServiceProviders.php [hunk 2]
```diff
@@ -64,4 +66,33 @@ private function resolveCommands()
...
+
+    private function printMethodChain($key, $array)
+    {
+        return array_key_exists($key, $array) ? "->$key($array[$key])" : '';
+    }
+
+    private function extractArguments($arguments)
+    {
+        list($url, $options) = explode(',', $arguments, 2);
+        $options = collect(explode(',', trim(trim(str_replace(['[', ']'], ['', ''], $options)), ','))) // remove array brackets, trim trailing comma
```
Comment (karam-mustafa) [inline src/Providers/YFrameServiceProviders.php]: please add the comments before each declaration, this help us keep the style ci clean

**Review by karam-mustafa — CHANGES_REQUESTED** [review]

File: src/Providers/YFrameServiceProviders.php (hunk 1 above)
Comment (RoduanKD) [reply src/Providers/YFrameServiceProviders.php:76]: it's already added!
image

File: src/Providers/YFrameServiceProviders.php (hunk 2 above)
Comment (RoduanKD) [reply src/Providers/YFrameServiceProviders.php]: also here
image

Comment (karam-mustafa) [reply src/Providers/YFrameServiceProviders.php]: Not actually, I was mean these comments before the options variable
Screenshot 2022-02-04 222901

**Review by karam-mustafa — CHANGES_REQUESTED** [review]

**Review by karam-mustafa — APPROVED** [review]
Done, Thanks a lot

**Pull request merged**